python/doc/theory/reliability_sensitivity/sensitivity_hsic.rst

```diff
@@ -0,0 +1,293 @@
+.. _sensitivity_hsic:
+
+Sensitivity analysis using Hilbert-Schmidt Indepencence Criterion (HSIC)
+-----------------------------------------------------------------------
```

Comment:
```suggestion
------------------------------------------------------------------------
```
I apologize, compiling on my machine did not raise any warning concerning this issue